tiledb/sm/crypto/test/unit_crypto.cc

```diff
@@ -29,16 +29,45 @@
  * @section DESCRIPTION
  *
  * Tests the `Crypto` class
+ *
+ * @section Test plan
+ *
+ * The random number generator is tested by generating two 64-byte buffers with
+ * cryptographically random data and checking that their content is not the
+ * same. The probability of having the same content is vanishingly small.
+ *
+ * TODO: add documentation for the AES-GCM tests.
```

Comment:
What to write here?

Reply:
Something like below: Where do the test vectors come from? What citation is appropriate?